apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -108,9 +108,13 @@ export class WorkflowQueueService {
     };
 
     const digestAdded = await this.addDigestJob(data, options);
+    const delayAdded = await this.addDelayJob(data, options);
     if (digestAdded) {
       return;
     }
+    if (delayAdded) {
+      return;
+    }
```

Comment:
```suggestion
    if (digestAdded || delayAdded) {
      return;
    }
```

Will this work aswell?